src/parser.ts

```diff
@@ -89,7 +89,7 @@ export class Parser {
 		commentMatchString += ")+([ ]*[a-z]|[:])+([^*/\\r\\n]*)";
 
 		// Find rows of comments matching pattern
-		let regEx = /(^|[ \t])(\/\*)+([\s\S]*?)(\*\/)/gm;
+		let regEx = /(^|[ \t])([\/\(]\*)+([\s\S]*?)(\*[\/\)])/gm;
```

Comment:
Could you explain this change to me? What's it fixing?

Reply:
This fixes multi line comment support for F#, as it uses `(*...*)` for multi line comment. If you don't agree with this I can remove this change, but then F# can't handle multi line comments

Reply:
Ah I see. I think multiline support for other languages needs a broader approach to make it consistent, so maybe leave this out for now. The rest of the multiline code works with JSDoc style so that's probably no good for anything else anyway

Reply:
Ok, removing it